.github/workflows/release_win.yml

```diff
@@ -46,6 +49,7 @@ jobs:
         python setup.py bdist_wheel
         Get-ChildItem -Path dist/*.whl | foreach {pip install --upgrade $_.fullname}
         pytest
+        # upload to PyPI automatically
```

Comment:
"actions/upload-artifact@v1" does not upload to pypi. Check : https://github.com/actions/upload-artifact
We want this action but the comment is wrong

Reply:
Yes... Sorry that I forgot to mention it is TODO